src/ios/ApptentiveStyles.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Instructions</key>
+	<string>This plist is for configuring styles that are difficult or impossible to target with UIAppearance. Use keys from ApptentiveStyleSheet.m and hex values for specifying colors.</string>
```

Comment:
Maybe link to our styling guide, if that would be useful?

Reply:
Done. Also updated the docs to include this stuff: https://learn.apptentive.com/knowledge-base/interface-customization-ios/#initializing-from-a-property-list